### DISCUSSION TITLE
[ENHANCEMENT] Add slower scan speed options

### DISCUSSION THREAD

**Discussion by ArisneR3** [Andrewthe13th/Inventory_Kamera#374] [Ideas or Feature requests]
Please add longer scan delay, genshin loads the inventory, characters etc sometimes too long and the whole scan breaks and you have to scan again till it finally works (kinda annoying with shit ton of artifacts)

**Comment by Cupcak3** [comment]
Hi there, [V1.2.5](https://github.com/Andrewthe13th/Inventory_Kamera/releases/tag/v1.2.5) has been released. Hopefully fixes in this version's new filters help you scan less items than you need to. Please let us know if they're resolved, thanks!

Additionally, I'll look into adding longer delays for lower-end PCs.

**Comment by ArisneR3** [comment]
Now the scanner closes itself during artifact scan. It also skips one row when scanning character items and materials. And some of the scans are incorrect too (wrong values). Everything is setup correctly (1280x720 res and whole game is visible). And actually instead of adding longer delays I suggest it to scan after taking all screenshots because I think the scanning makes game work slower and more unstable. It's not a comfortable solution but its all about survival on low end I guess xD.
[InventoryKamera.log](https://github.com/Andrewthe13th/Inventory_Kamera/files/8697533/InventoryKamera.log)

**Comment by Cupcak3** [comment]
Newer versions should have a slow scan option which should be enough for low-end computers. If that is still too fast... I guess I could make it even slower?

**Comment by ogeraisi** [comment]
How about replacing the speed slider with a textbox for the desired ms value?

**Comment by Cupcak3** [comment]
Letting users set delays manually would be more trouble than it's worth to debug. There are also varying specific delays based on the selection from the menu.
